App.js

```diff
@@ -63,7 +65,11 @@ const AppStack = createBottomTabNavigator(
         const { routeName } = navigation.state;
         let iconName;
         if (routeName === "Home") {
-          iconName = "home";
```

Comment:
Why replacing "home" icon with a new one?

Reply:
fixed it 👍🏻